communities/plymouth-university/community-assessment.md

```diff
@@ -0,0 +1,122 @@
+
+GDSoc is made up of computer scientists and aritsts, there is a large imbalance between engineers and designers within the society. However as it is often run by members of the Computing & Games Dev course it's content is often more tailored towards engineers.
+
+I'd love to get more designers involved and also some first time programmers who aren't on comp sci courses!
+
+### Is your community open to other topics, or are their interests very specific?
+
+If we take the only tech society on campus then the interest is specifically games development, but the university as a whole has many computing and comp sci course flavours who have a wide range of interests from web to AI and all sorts.
+
+## Goals
+
+### Why do people get involved in the community?
+
+One of the biggest across the university seems to be obtaining a 1 year internship for their "sandwich year", or for getting a job after their degree. Due to the games students being most involved, they also seem to get move involved when the content is likely to contribute towards current assignments.
+
+The one of the problem with GDSoc is that most of the Games Dev course content is based on Unity3D, however this is quite an uncommon technology to use outside of mobile or indie games. Web content seems to cause heavy disengagement from GDSoc members as there seems to be a level of elitisim that attempts to stray away from web based games.
+
+### What activities and goals can we set for the community based on existing goals?
+
+One of the goals we could look towards is trying to find to put on technical talks by experts which relate heavily to lecture content. This has been hard due to the lack of industry in and around Plymouth but it should be a goal to strive towards.
+
+## Location
+
+Due to the nature of the community, the key location for all events and meets is the Plymouth University campus. Any room on campus can be booked out by a students union registered society, this means the GDSoc can book rooms with 2 weeks notice.
+
+The bookable space is very flexible ranging from high spec computer labs which can accomodate up to around 30 or 40 people, over to large lecture theatres which can seat around 200-300. These spaces can be booked for anytime they're not already booked between the hours of 9am and 6pm. After that we need to find a different space.
+
+Lecture theatres are generally very good for speaker based events as the chairs are often quite comfortable and suitable for longer periods of time. They also fit fire restrictions for large groups of people. The computer labs are perfect for small workshops where people can log on and make use of the computers. The chairs in these rooms are also quite comfortable but not always great for talk focused sessions.
+
+There are many pubs with bigger spaces which can be booked for events and these tend to fit up to around 40 people. At the same time smaller meets can happen in the many cafes or pubs around the city and campus.
```

Comment:
As TermiSoc went, so did Voodoo Lounge. It's now a new bar that I've yet to try though!